app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -75,7 +73,6 @@ public class SubscriptionFragment extends Fragment
         SubscriptionsRecyclerAdapter.OnSelectModeListener {
     public static final String TAG = "SubscriptionFragment";
     private static final String PREFS = "SubscriptionFragment";
-    private static final String PREF_NUM_COLUMNS = "columns";
```

Comment:
Instead of moving the setting to be globally, I would add a field to the Adapter that contains the number of columns. I'm trying to reduce the dependencies of various parts of the app on `UserPreferences.java`. (this also ensures that the setting that users selected does not get reset on upgrade)